src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -2339,4 +2338,65 @@ private function isProxyCandidate(Definition $definition, ?bool &$asGhostObject,
 
         return $this->getProxyDumper()->isProxyCandidate($definition, $asGhostObject, $id) ? $definition : null;
     }
+
+    /**
+     * Removes comments from a PHP source string.
+     *
+     * We don't use the PHP php_strip_whitespace() function
+     * as we want the content to be readable and well-formatted.
+     */
+    private function stripComments(string $source): string
```

Comment:
I don't know how to test because method is now private. I think we should re-use [same dataprovider](https://github.com/symfony/symfony/pull/51712/files#diff-d85a8adbc4a569ab97ca42339d9bd47d41025f2af53cea0c537d5ed55276625bR271).

Can I use ReflectionClass or what is the best practice in this repository ?

Reply:
Testing via reflection should be fine, I think.

Reply:
I suggest keeping that method static as it does not depend on anything. It will make it easier to test it (not need to create a fake container to create a dumper)